### ISSUE TITLE
Incorrect ejabberd versione: "ejabberd 0.0 is running in that node"

### ISSUE THREAD

**Issue by Suvitruf** [processone/ejabberd#920]
`ejabberdctl status` shows incorrect ejabberd version:

     root@chat1:~# ejabberdctl status
     The node 'ejabberd@x.x.x.x' is started with status: started
     ejabberd 0.0 is running in that node

I've builded ejabberd version 16.01 from sources.

Same if I'm trying to get status via service:

     root@chat1:~# service ejabberd status
     Getting ejabberd status...
     The node 'ejabberd@x.x.x.x' is started with status: started
     ejabberd 0.0 is running in that node

**Comment by badlop** [comment]
The ejabberd version number is read when ejabberd is starting from the file 
> /lib/ejabberd-16.01/ebin/ejabberd.app

It should say something like:
```
{application,ejabberd,
             [{description,"ejabberd"},
              {vsn,"16.01"},
...
```

That version number is also mentioned at server start in the logs:

```
18:37:55.280 [info] Application lager started on node ejabberd@localhost
18:37:55.294 [info] Application crypto started on node ejabberd@localhost
...
18:37:59.254 [info] Reusing listening port for {5280,{0,0,0,0},tcp}
18:37:59.255 [info] ejabberd 16.01 is started in the node ejabberd@localhost
```

I tried ejabberd git and ejabberd 16.01 source code in Debian, and the version number is always correctly set. How did you get ejabberd sources, and how did you compile them?

**Comment by Suvitruf** [comment]
https://github.com/processone/ejabberd/archive/16.01.tar.gz

     ./autogen.sh
     ./configure --enable-all
     make
     make install

**Comment by badlop** [comment]
That happens because that package doesn't use git, and the code can't get the version number from git. That is done in this line:
https://github.com/processone/ejabberd/blob/master/configure.ac#L5

This problem doesn't appear in the published source package, because that call to git is not present, instead the proper version number is provided:
> AC_INIT(ejabberd, 16.01, ejabberd@process-one.net, ejabberd)

So, you are right, that problem appears when using that package built by github, but that isn't a problem, since that package is not supported. Either use a git cloned repository, or the published source package, or life with that 0.0 version number, or replace it manually in your 'configure' script after running autogen.sh.
